### PR TITLE
full audit managed ops server config

### PR DESCRIPTION
**Pull request by stephamon**
apply SME feedback

### CODE REVIEW

**Comment by stephamon** [comment]
Hey @cyrichardson can you give this a look please?

**Comment by cyrichardson** [comment]
Will do, @stephamon.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b9b8b59930/how-to/)

**Comment by cyrichardson** [comment]
@stephamon Some suggested changes here: 
[cloud-server-configuration-options_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2275505/cloud-server-configuration-options_cyrEdits.docx)

File: content/managed-operations/cloud-server-configuration-options.md [hunk 1]
```diff
@@ -103,58 +102,56 @@ Depending upon your service level, these options are available for the
...
 
-The following example shows the build option for a Chef server
-management.
+The following example shows the build option for a Managed Infrastructure server:
 
     nova boot --flavor 2 --image [image] --meta build_config=core www.example.com
 
-The following example shows the build option for building a file server
-with a Rackspace user name and password only and backup ability.
+The following example shows the build option for building Managed Operations server with a Rackspace user name and password only and backup ability:
```
Comment (RackerKate) [inline content/managed-operations/cloud-server-configuration-options.md]: "... building a Managed Operations server..."

File: content/managed-operations/cloud-server-configuration-options.md [hunk 2]
```diff
@@ -103,58 +102,56 @@ Depending upon your service level, these options are available for the
...
--   Build Error
--   Authentication Error
-
-When the metadata key is set to **Complete**, you can begin your
+configuration to complete before performing your configuration management. For
+standard and workload-optimized cloud servers only, you can obtain the Managed
+Operations or Managed Infrastructure post-build automation status via the
+Cloud Servers API. Look for the metadata key value  
+`informationrax_service_level_automation`. Depending on the current status of
+the Managed Cloud post-build automation process, the metadata key will have
```
Comment (RackerKate) [inline content/managed-operations/cloud-server-configuration-options.md]: Probably not a big deal in this context, bug suggest "key has".

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1dad4948fa/how-to/).

**Pull request merged**